### ISSUE TITLE
Google events from specific domain not syncing

### ISSUE THREAD

**Issue by LesegoT** [phw198/OutlookGoogleCalendarSync#740]
[OGcalsync.log](https://github.com/phw198/OutlookGoogleCalendarSync/files/3212492/OGcalsync.log)

:memo: Use the below template to report your issue.  
:bulb: Upgrade to the latest alpha release to check if the problem is already fixed.  
:paperclip: To aid investigation, provide your OGcalsync.log file ([see instructions](https://github.com/phw198/OutlookGoogleCalendarSync/wiki/Reporting-problems#locating-your-ogcalsynclog-file)).  

----

**OGCS Version**: v2.7.9.0  
**Installed or Portable**: Installed

**Problem Description**  

Google calendar events sent from a specific domain (unisa.ac.za) are not syncing to Outlook, although they used to sync before. I suspect this happened with the update to 2.7.9 but cannot say for sure. I tried manually creating an event in Google and this syncs fine. I have also sent a meeting request from a different email address to my Gmail, and after accepting it in Gmail, it also syncs fine. It is just the new ones from unisa.ac.za that do not sync.

**Steps to Reproduce the Issue**  
1. Accept a meeting request from *@unisa.ac.za. The notification is sent to the requester and the item appears in Google Calendar (both on the web client as well as in Windows 10 Calendar).
2. Run a bi-directional sync. The items do not even appear in the sync window.

**Any other information**  
\<Update here>

**Comment by LesegoT** [comment]
[OGcalsync.log](https://github.com/phw198/OutlookGoogleCalendarSync/files/3265135/OGcalsync.log)

**Update:**
I have updated to 2.8.0.0 and the issue is still recurring. I have noticed that this issue occurs when I accept the Gmail meeting requests in the Windows 10 Calendar app, and also on Gmail through the web. It does not happen when I click on "View on Google Calendar" and accept it from there.

**Comment by phw198** [comment]
So are you saying _regardless of the email domain_ an invitation comes from, if you accept it via Windows 10 Calendar or Gmail, is doesn't get synced by OGCS?

I've just sent an invitation to my Gmail account and accepted in Gmail, it's gone in to my default Google calendar and OGCS has synced it fine. Just to check the simple things - the invitation does get placed into the _same_ calendar that you are syncing with OGCS?

Having accepted an invite in both ways (one that OGCS syncs and one that doesn't), you could try exporting the calendar, which will give you an `.ics` file - then have a look through the relevant entries to see if there are any differences that might be causing the problem...

**Comment by phw198** [comment]
I just noticed this settings option in Google calendar - I wonder if this might be part of the issue:
![image](https://user-images.githubusercontent.com/11843015/59153401-79fab100-8a50-11e9-917a-0383fe4a6d06.png)

**Comment by LesegoT** [comment]
So far I have only observed invitations that come via Gmail from that one specific domain, and that I accept in the Windows 10 Calendar or in Gmail do not get synced. Only when I say "View on Google Calendar" and accept from there does it then sync. I tried sending myself invitations to Gmail from my other email addresses, and those do sync even when I accept them in the W10 Calendar. It seems the problem might be my W10 Calendar client not sending the acceptance to Google Calendar.

I have also turned on the "Automatically add events..", but I have found that just adds events it picks up in the actual email, for example when there is a mail saying see doctor on Thursday, then it automatically adds that, but not meeting invitations.

I will try the calendar export when I next receive an invitation from that domain.

**Comment by LesegoT** [comment]
I have now reproduced this issue as follows:

1. Received a meeting invitation from someone @unisa.ac.za in W10 Mail.
2. Accepted the invitation in W10 Mail
[OGcalsync(after W10 acceptance).log](https://github.com/phw198/OutlookGoogleCalendarSync/files/3280345/OGcalsync.after.W10.acceptance.log)
. The calendar entry is created in the W10 Calendar, and the email invitation deleted.
3. Checked Google Calendar. The meeting is not there, and the email invitation is in the Gmail Bin (so that part synced)
4. Ran OGCS, the new meeting did not sync.
5. Went back to Gmail Bin and clicked Yes on the meeting invitation. The meeting now appears in Google Calendar.
6. Ran OGCS sync again and the meeting now syncs.

Again this only happens with meeting invitations from that one domain. So it appears that the problem is the W10 Calendar not updating the Google Calendar. 
[OGcalsync.log](https://github.com/phw198/OutlookGoogleCalendarSync/files/3280356/OGcalsync.log)

I suppose I can live with this workaround because there aren't that many invitations I receive from that domain.
